bokehjs/src/coffee/core/layout/layout_canvas.coffee

```diff
@@ -0,0 +1,49 @@
+_ = require "underscore"
+
+{Variable, EQ, GE, Strength}  = require "./solver"
+Model = require "../../model"
+p = require "../properties"
+
+class LayoutCanvas extends Model
+  type: 'LayoutCanvas'
+
+  @internal {
```

Comment:
I'm not entirely sure what an internal class is, but I'm guessing yes - it's not something we are going to expose to users. How do I mark it as such?